dashboard/app/models/pd/enrollment.rb

```diff
@@ -49,6 +49,15 @@ class Pd::Enrollment < ActiveRecord::Base
   validate :validate_school_name, unless: :created_before_school_info?
   validates_presence_of :school_info, unless: :created_before_school_info?
 
+  after_create :enroll_in_corresponding_online_learning
+
+  WORKSHOP_COURSE_ONLINE_LEARNING_MAPPING = Hash[
+    Pd::Workshop::COURSE_CSP, 'CSP Support',
+    Pd::Workshop::COURSE_ECS, 'ECS Support',
+    Pd::Workshop::COURSE_CS_IN_A, 'CS in Algebra Support',
+    Pd::Workshop::COURSE_CS_IN_S, 'CS in Science Support'
+  ]
```

Comment:
This mapping seems more related to workshop than enrollment, and should go with the other workshop constants in `workshop.rb`. It can be exposed via something like `workshop.associated_online_course`

Also why `Hash[...]` instead of `{...}`? And please `.freeze` the hash. See [SECTION_TYPE_MAP](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/pd/workshop.rb#L95-L104) for an example.